mlflow/server/js/src/components/CompareRunView.js

```diff
@@ -1,9 +1,9 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import { getParams, getRunInfo } from '../reducers/Reducers';
+import {getExperiment, getParams, getRunInfo} from '../reducers/Reducers';
```

Comment:
nit: `{getExperiment, getParams, getRunInfo}` -> `{ getExperiment, getParams, getRunInfo }`. We should probably have a standard way of setting up common IDEs.